onnxruntime/core/language_interop_ops/torch/refcount_tracker.h

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+#include <Python.h>
+#include "core/platform/env.h"
+
+namespace onnxruntime {
+namespace language_interop_ops {
+namespace torch {
+#ifndef NDEBUG
+
+using AddressInfos = std::unordered_map<void*, std::vector<std::string>>;
+
+class RefCountTracker {
+ public:
+  enum ObjCategory {
+    ForwardArgs,
+    ReturnValues,
```

Comment:
i.e. the returns from the forward call?  Do we need separate categories for the inputs to the backward call, and returns from the backward call?

Reply:
I think the naming ForwardArgs is not good one. Actually I think we just need a python_call_argus, returned values and augograd context for monitoring. 

Splitting these into forward and backward is not that necessaryt because from the logging, we can easily figure out whether it is forward call or backward call.